src/sidebar/templates/login-control.html

```diff
@@ -20,12 +20,8 @@
   </a>
   <ul class="dropdown-menu pull-right" role="menu">
     <li class="dropdown-menu__row" ng-if="vm.auth.status === 'logged-in'">
-      <span ng-if="vm.isThirdPartyUser()"
-            class="dropdown-menu__link dropdown-menu__link--disabled">
-        {{vm.auth.username}}</span>
-      <a ng-if="!vm.isThirdPartyUser()"
-         href="{{vm.serviceUrl('user',{user: vm.auth.username})}}"
-         class="dropdown-menu__link"
+      <a ng-click="vm.showProfile()"
+         class="dropdown-menu__link dropdown-menu__user-profile-btn"
```

Comment:
Now that I think about it, we should stick with the disabled user profile link (as previously) if we are on a publisher group site but no `onProfileRequested` is provided. I'll fix this